hi-ml-histopathology/src/histopathology/datasets/panda_dataset.py

```diff
@@ -115,9 +118,11 @@ def __call__(self, data: Dict) -> Dict:
         # but relative region size in pixels at the chosen level
         scale = mask_obj.resolutions['level_downsamples'][self.level]
         scaled_bbox = level0_bbox / scale
-        get_data_kwargs = dict(location=(level0_bbox.x, level0_bbox.y),
-                               size=(scaled_bbox.w, scaled_bbox.h),
-                               level=self.level)
+        get_data_kwargs = dict(
+            location=(level0_bbox.y, level0_bbox.x),
```

Comment:
Note the swapped x and y here!

Reply:
Interesting we never realized we were swapping the coordinates - is it something we can test?

Reply:
> Interesting we never realized we were swapping the coordinates - is it something we can test?

I guess we could write a test for this using a synthetic WSI. Can we write WSIs?

Reply:
Yep, you can use [MockPandaSlidesGenerator](https://github.com/microsoft/hi-ml/blob/39911d217c919d8213ad36c9c776f69369d98509/hi-ml-histopathology/testhisto/testhisto/mocks/slides_generator.py#L24)

Reply:
Thanks, @kenza-bouzid! I'll take a look.

Reply:
I would prioritise merging this in case it fixes the issue reported by @harshita-s, then we can think of adding some tests if you like. Do you agree?